### PR TITLE
Revert "Implemented Secure Hashing Algorithms (#828)"

### PR DESCRIPTION
**Pull request by DanVanAtta**
This reverts commit 3acc3d7dd39a183f144c2f279326ec2650e6b008.

Getting these errors: https://gist.github.com/DanVanAtta/04932362466fe1f7390f0ca7305fbcdf, on linux, latest master.

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/857)

### CODE REVIEW

**Comment by DanVanAtta** [comment]
@RoiEXLab  do you have access to a linux machine? Virtual box could be one route if you do not. This looks OS specific, and a wider implication is that games running on linux  would not be able to join the lobby.

**Comment by RoiEXLab** [comment]
@DanVanAtta, yes I have, if I get my virtualbox running eventually... But the question I'm asking myself, is why there are no failures on the travis build server, because this server is running Linux...

**Comment by RoiEXLab** [comment]
And I would appreciate it, if you'd only revert the getMacAdress method if needed... All the other stuff is not used currently... 
When reverted the method, please remove the ugly cross-java-version code, which only gets executed at java 6+

**Comment by RoiEXLab** [comment]
TLDR, I don't think that this is os specific, since the build server has no problems... 
I think this is system related depending on your (virtual) network card configuration.
Or java has a bug...

**Comment by DanVanAtta** [comment]
#828 does a partial revert as requested.

I tested on my desktop system, tests passed fine there - but not on the laptop. This explains then why the Travis build is passing, problem is perhaps related to WIFI vs ethernet connections.
